### ISSUE TITLE
`emmylua_check` doesn't appear to analyse library files.

### ISSUE THREAD

**Issue by lewis6991** [CppCXY/emmylua-analyzer-rust#329]
With https://github.com/lewis6991/gitsigns.nvim/pull/1251 and running `make emmylua_check` gives false warnings like:

```
[undefined-field] Warning:
     ╭─[ gitsigns/attach.lua:150:19 ]
     │
 150 │   local file = uv.fs_realpath(api.nvim_buf_get_name(bufnr))
     │                   ─────┬─────
     │                        ╰─────── Undefined field `fs_realpath`.
─────╯
```

Which suggests it is not analysing the `luv` directory.

Using an absolute path in `.luarc.json` fixes the issue.
